From 559e0909bb554f32e3ef7b47bba2077db5159b43 Mon Sep 17 00:00:00 2001
From: Zhengchao Shao <shaozhengchao@huawei.com>
Date: Fri, 7 Apr 2023 14:23:06 +0800
Subject: [PATCH 916/921] [Huawei] bpf: tcp: use space in front of the comment

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 188649

-------------------------------

Use 1 blank space(' ') instead of TAB('\t') between the right comment
and the previous code.

Link: https://clouddragon.huawei.com/codecheck/project/g58ed40cf6b794038a0d206e00bc8a7e9/codecheck/task/de30fa401960491e860a5aa7df04cd31/defects
Signed-off-by: Zhengchao Shao <shaozhengchao@huawei.com>
---
 net/ipv4/bpf_tcp_ca.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/ipv4/bpf_tcp_ca.c b/net/ipv4/bpf_tcp_ca.c
index 94863facecd7..27112974ba09 100644
--- a/net/ipv4/bpf_tcp_ca.c
+++ b/net/ipv4/bpf_tcp_ca.c
@@ -214,9 +214,9 @@ BTF_ID(func, bbr_ssthresh)
 BTF_ID(func, bbr_min_tso_segs)
 BTF_ID(func, bbr_set_state)
 #endif
-#endif	/* CONFIG_PAHOLE_VERSION */
-#endif  /* CONFIG_DYNAMIC_FTRACE */
-#endif	/* CONFIG_X86 */
+#endif /* CONFIG_PAHOLE_VERSION */
+#endif /* CONFIG_DYNAMIC_FTRACE */
+#endif /* CONFIG_X86 */
 BTF_SET_END(bpf_tcp_ca_check_kfunc_ids)
 
 static const struct btf_kfunc_id_set bpf_tcp_ca_kfunc_set = {
-- 
2.17.1

